tests/bench.py

```diff
@@ -227,3 +227,14 @@ def test_image_new_property_performance(benchmark):
     data = BytesIO(get_data_from_url(url, name=name))
 
     benchmark(image_new_property, data)
+
+
+def image_extraction(reader):
+    for page in reader.pages:
+        list(page.images)
+
```

Comment:
Could you please add another empty line here to be consistent?

Reply:
Done